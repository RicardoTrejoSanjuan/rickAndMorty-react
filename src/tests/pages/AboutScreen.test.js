import React from 'react';
const { shallow } = require("enzyme");
const { default: AboutScreen } = require("../../pages/AboutScreen");

describe('Test en <AboutScreen />', () => {

    const title = 'Ricardo Trejo Sanjuan';
    const wrapper = shallow(<AboutScreen />);
    
    test('Dede mostrar el componente', () => {
        expect(wrapper).toMatchSnapshot();
    });

    test('Debe contener un <h3>', () => {
        const h3 = wrapper.find('h3');
        expect(h3.text()).toBe(title);
    });
});