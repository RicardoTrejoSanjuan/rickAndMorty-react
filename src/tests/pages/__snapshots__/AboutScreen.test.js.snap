// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test en <AboutScreen /> Dede mostrar el componente 1`] = `
<Fragment>
  <div
    className="container p-4"
  >
    <h3>
      Ricardo Trejo Sanjuan
    </h3>
    <p />
    <p
      className="text-justify"
    >
      Curious and passionate person with ease to express my opinion and thoughts about any topic. I consider myself as a great admirer of IT and software development, as well as the impact it has in our daily lives. I focus my formation like Full Stack developer. My analytic and self-taught aptitudes allow me to solve problems efficiently and turn them into a software solution.
    </p>
    <h4>
      About Me :D
    </h4>
    <pre
      className="header-inner"
    >
      <code
        className="hljs"
      >
        {
   "pronouns": "He",
   "code": [
      "Javascript",
      "Typescript",
      "Go",
      "Dart",
      "Java",
      "Bash",
      "HTML"
   ],
   "askMeAbout": [
      "FrontEnd Dev",
      "BackEnd Dev",
      "FullStack Dev",
      "App Dev"
   ],
   "technologies": {
      "editors": [
         "VSCode",
         "Visual Studio"
      ],
      "mobileApp": [
         "Flutter",
         "Ionic",
         "Android",
         "iOS",
         "Cordova"
      ],
      "frontEnd": {
         "js": [
            "ES6",
            "Angular",
            "React",
            "AngularJs",
            "npm",
            "grunt",
            "gulp",
            "webpack",
            "storybook"
         ],
         "css": [
            "Bootstrap",
            "Bulma",
            "Angular Material",
            "SASS",
            "FlexBox"
         ]
      },
      "backEnd": {
         "js": [
            "NodeJs",
            "express",
            "got",
            "handlebars"
         ],
         "go": [
            "mux"
         ],
         "java": [
            "Spring Boot"
         ]
      },
      "testing": {
         "js": [
            "Jest",
            "Jasmine",
            "Karma",
            "MochaJs"
         ]
      },
      "devOps": [
         "Openshift",
         "Docker",
         "GitlabCI",
         "Jenkins"
      ],
      "databases": [
         "PostgreSQL",
         "MongoDB",
         "MySql",
         "SQL Server",
         "sqlite"
      ],
      "misc": [
         "git",
         "OpenApi",
         "WebSockets",
         "Postman"
      ]
   }
}
      </code>
    </pre>
  </div>
</Fragment>
`;
